src/features/common/data/algo-client.ts

```diff
@@ -17,13 +17,19 @@ const shouldCreateKmdClient = (config: NetworkConfig) => {
 // Init the network config from local storage
 const networkConfig = settingsStore.get(networkConfigAtom)
 export let indexer = ClientManager.getIndexerClient(networkConfig.indexer)
+
 export let algod = ClientManager.getAlgodClient(networkConfig.algod)
```

Comment:
Should we set this on indexer and kmd as well, just in case?

Reply:
I think that's a good idea, let me do it.